webtau-feature-testing/examples/listeners/MetadataValidationTestListener.groovy

```diff
@@ -6,7 +6,7 @@ import org.testingisdocumenting.webtau.reporter.WebTauTest
 class MetadataValidationTestListener implements TestListener {
     @Override
     void afterLastTestStatement(WebTauTest test) {
-        if (test.hasSteps() && !test.metadata.has('owner')) {
+        if (!test.isSynthetic() && !test.metadata.has('owner')) {
```

Comment:
as @tsiq-nick mentioned, relying on steps is not future-proof

Reply:
btw I am not using that account anymore, I use my personal one.

Reply:
whoops, sorry